packages/table/src/store/index.ts

```diff
@@ -150,15 +152,15 @@ function useStore(): Store {
       instance.store.updateCurrentRow(row)
     },
   }
-  const commit = function (name, ...args) {
+  const commit = function(name, ...args) {
     const mutations = instance.store.mutations
     if (mutations[name]) {
       mutations[name].apply(instance, [instance.store.states].concat(args))
     } else {
       throw new Error(`Action not found: ${name}`)
     }
   }
-  const updateTableScrollY = function () {
+  const updateTableScrollY = function() {
     nextTick(instance.layout.updateScrollY.apply(instance.layout))
```

Comment:
Use 
```ts
nextTick(() => instance.layout.updateScrollY.apply(instance.layout))
```

Reply:
@justwiner  can you address this comment?

Reply:
yeah, it's right